test/react-web/client/graphql/queries.tsx

```diff
@@ -1209,7 +1232,7 @@ describe('queries', () => {
           this.props.data.updateQuery();
           done(new Error('should have thrown'))
         } catch (e) {
-          expect(e).to.match(/Invariant Violation:/);
+          expect(e).to.match(/ObservableQuery with this id doesn't exist:/);
```

Comment:
There's no longer an invariant check here because it goes straight through to AC, but I think the error is OK? I'm not sure. We can definitely improve the error at AC's end anyway.